completion/_autocomplete.history_lines

```diff
@@ -5,13 +5,12 @@ _autocomplete.history_lines() {
   _tags $tag
   _tags && _requested $tag ||
     return 1
-  echo "hello"
   local -aU events=()
   local -i max_lines=$(( min( 16, HISTNO, LINES - BUFFERLINES ) ))
   if [[ -z $BUFFER ]]; then
     local query='<-16>'; [[ $curcontext == *-forward:* ]] ||
       query="<$(( HISTNO - max_lines ))->"
-    print -v events -f '%s:%s' "${(kv@)history[(I)$~query]}"
+    print -v events -f '%s:%s' "${(ukv@)history[(I)$~query]}"
```

Comment:
That's not going to work. It will eliminate duplicate values, but will leave their keys in place. What you can do is 
1. reverse sort the associative array, so you get a map of values to keys and
1. apply uniqueness to the values to eliminate duplicates, then 
1. look up the key for each value in the reverse map.

Reply:
Yeah I realized that after I pushed... I'll try those steps.

Reply:
Actually, I just realized it's simpler than that: Step 1 will already eliminate all duplicates, because associative arrays don't allow duplicate keys. So, just reverse sort and then reverse sort again and you should be pretty much done. 🙂

Reply:
Ok cool. I'll try to have a new commit tomorrow. I got pulled away to other stuff the last couple days.